modules/memstore_redis/main.tf

```diff
@@ -1,8 +1,9 @@
 terraform {
   required_providers {
     google = {
-      source  = "hashicorp/google"
-      version = "~> 4.0"
+      source = "hashicorp/google"
+      # because v4.16.0 is not google-beta for replica_count
```

Comment:
Not sure what this sentence means?

Reply:
I mean `replica_count` is still in beta (`google-beta`) until `4.15.0`, since `4.16.0` you can use `replica_count` in `google`

Reply:
```
# because `replica_count` and `read_replicas_mode` require v4.16.0 for `hashicorp/google`, before 4.16.0 need `hashicorp/google-beta`
```

@MXfive better?